packages/components/pie-textarea/src/textarea.scss

```diff
@@ -21,6 +21,7 @@
   line-height: 0; // Remove once there is text outside the textarea
 
   textarea {
+    width: 100%;
```

Comment:
I am curious why this is necessary, and if the element should be `display: block`/`inline-block` instead?

Reply:
<img width="724" alt="Screenshot 2024-07-26 at 16 46 49" src="https://github.com/user-attachments/assets/0fdf46c9-f224-4bb8-b818-1ac75d7a1fee">

display: block/inline-block doesn't work, you can check this in pie-aperture form